### ISSUE TITLE
Request: Porphyry House System

### ISSUE THREAD

**Issue by imakatman** [0xStarcat/CircularNatalHoroscopeJS#37]
Hello,

Thank you for making this project open-source! I wanted to put in a request for the Porphyry house system. It seems like its simplicity of setting the 0 degree of the ascendant angle is gaining popularity and I'd like to have the option to use it in my own app that I am creating.

Thank you very much!
Hope

**Comment by charliejlevine** [comment]
@imakatman Agreed, this would be great! It's the one used by Co-Star: https://www.costarastrology.com/how-does-astrology-work/house-systems. @0xStarcat, do you have a reference for adding Porphyry House System to this project?
